bin/keycloak-httpd-client-install

```diff
@@ -943,6 +966,11 @@ def main():
                        default='/etc/httpd',
                        help='Template location')
 
+    group.add_argument('--module',
+                        required=True,
```

Comment:
This shouldn't be required because it breaks backwards compatibility, instead it should default to mod_auth_mellon.